sympy/sets/sets.py

```diff
@@ -706,6 +706,19 @@ def _union(self, other):
 
     @property
     def sets(self):
+        """
+        Returns the arguments of 'self', or in the other words, the factors
```

Comment:
'the' factors? This should better be `a tuple of factors`. Are the factors of cartesian product in maths unique?